buildSrc/src/main/kotlin/io/spine/internal/gradle/publish/Publications.kt

```diff
@@ -198,7 +203,7 @@ internal class StandardJavaPublicationHandler(
  *
  * Such publications should be treated differently than [StandardJavaPublicationHandler],
  * which is <em>created</em> for a module. Instead, since the publications are already declared,
- * this class only [assigns maven coordinates][assignMavenCoordinates].
+ * this class only [assigns maven coordinates][copyProjectAttributes].
```

Comment:
I see the reference has changed, but did you intend to leave the link text the same?